charts/fluent-operator/templates/fluent-operator-deployment.yaml

```diff
@@ -85,5 +88,5 @@ spec:
       serviceAccountName: fluent-operator
       {{- if .Values.operator.imagePullSecrets }}
       imagePullSecrets:
-      {{ toYaml .Values.operator.imagePullSecrets | indent 8 }}
+      {{- toYaml .Values.operator.imagePullSecrets | nindent 8 }}
```

Comment:
```suggestion
      {{- toYaml .Values.operator.imagePullSecrets | indent 8 }}
```

Reply:
nindent adds a new line first, so if using nindent we can write code as below:
```yaml
imagePullSecrets:  {{- toYaml .Values.operator.imagePullSecrets | nindent 8 }}
```
if the template text in already in a new line, then the original indent is ok:
```yaml
imagePullSecrets:
{{- toYaml .Values.operator.imagePullSecrets | indent 8 }}
```

Reply:
oh yeah you are right about this one :)

Reply:
second option will break as soon as there is more than one imagePullSecret I think
it will produce output such as:
```
      imagePullSecrets:
              secret0: value0
        secret1: value1
```
Explanation: secret0 will be indented by a total of 6+8=14 spaces because the templating statement is intended already by 6 spaces and indent 8 will add another 8 but for the second secret it will be only intended by 8+0=8 spaces in total. 

However your top solution(this one here below) should work no matter if there is one imagePullSecret or multiple but so does mine.
```
      imagePullSecrets:  {{- toYaml .Values.operator.imagePullSecrets | nindent 8 }}
```